docs/api/process.md

```diff
@@ -170,6 +171,18 @@ Returns `Object`:
 
 Returns an object with V8 heap statistics. Note that all statistics are reported in Kilobytes.
 
+### `process.getBlinkMemoryInfo()`
```

Comment:
Can you confirm this API works in the main process?  This API is living in `common/` in our code but the `TotalAllocatedObjectSize` APIs are `blink/renderer`

Reply:
it does work, it just returns zeros